.github/workflows/ci.yml

```diff
@@ -16,37 +16,19 @@ jobs:
       matrix:
         version:
           - '1.10'
-          - '~1.11.0-0'
+          - '1.11'
         os:
           - ubuntu-latest
           - macOS-latest
           - windows-latest
-        arch:
-          - x64
-        exclude:
-          - version: '~1.11.0-0'
-            os: macOS-latest # JET crashes on one unit test and hangs on another
-          - version: '~1.11.0-0'
-            os: windows-latest # JET crashes on 3 unit tests
     steps:
       - uses: actions/checkout@v4
       - uses: julia-actions/setup-julia@v1
         with:
           version: ${{ matrix.version }}
-          arch: ${{ matrix.arch }}
-      - uses: actions/cache@v1
-        env:
-          cache-name: cache-artifacts
-        with:
-          path: ~/.julia/artifacts
-          key: ${{ runner.os }}-test-${{ env.cache-name }}-${{ hashFiles('**/Project.toml') }}
-          restore-keys: |
-            ${{ runner.os }}-test-${{ env.cache-name }}-
-            ${{ runner.os }}-test-
-            ${{ runner.os }}-
-      - uses: julia-actions/julia-buildpkg@v1
-      - uses: julia-actions/julia-runtest@v1
-      - uses: julia-actions/julia-processcoverage@v1
-      - uses: codecov/codecov-action@v1
-        with:
-          file: lcov.info
+      - name: Add TestEnv
+        run: julia -e 'using Pkg; Pkg.add("TestEnv")'
+      - name: Instantiate project
+        run: julia --project -e 'using Pkg; Pkg.instantiate()'
+      - name: Run tests
+        run: julia --project -e 'using TestEnv; TestEnv.activate(); include("test/runtests.jl")'
```

Comment:
Maybe add a comment on why you are doing this instead of just using the runtest action

Reply:
I think I can switch this back to the runtest action if I just turn off checkbounds. We don't need checkbounds anyway since all iterators in this test suite are stack allocated.

Reply:
It turns out that switching off `check_bounds` is insufficient. In order to reduce the time required for this workflow, I've also had to disable coverage and remove the test harness wrapper added by the `julia-runtest` action. I simplified this workflow to call `Pkg.test`, and I've added a comment.